stylesheets/style.css

```diff
@@ -5,3 +5,222 @@ main-titles dark-grey: #2C303F;
 paragraph grey: #5b5e6d;
 nav grey: #5b5e6d;
 */
+
+/* ---- reset ---- */
+
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0 0px;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: white;
+  border-bottom: lightgrey 1px solid;
+  position: fixed;
+  width: 100%;
+}
+
+#logo {
+  height: 50px;
+  margin: 5px 20px;
+}
+
+nav ul{
+  display: none;
+}
+
+#menu {
+  margin: 10px;
+}
+.bar-menu {
```

Comment:
Try to be consistent with the format. If you put a blank space between the rules, put it all the time